tests/test_autograd.py

```diff
@@ -4,6 +4,7 @@
 import torch
 
 import bitsandbytes as bnb
+from bitsandbytes.cextension import HIP_ENVIRONMENT
```

Comment:
If this is run on NV, will this fail ?

Reply:
Yes, I think so, thanks for pointing it out @amathews-amd. I will add a default value.

Reply:
Fixed it here: https://github.com/ROCmSoftwarePlatform/bitsandbytes/pull/1/commits/28b80564b7efbec22eba9800ac4b83ee745a34c1